### PR TITLE
Disable logging for mobile

### PR DESCRIPTION
**Pull request by bhardwajaditya**

### CODE REVIEW

**Comment by ear-dev** [comment]
@bhardwajaditya This is not working for me when I embed the widget into a web page.  It only seems to work from the /livechat url directly.  Maybe this resolves to true in the embedded widget case?  (window.matchMedia('(max-width: 500px)').matches) {

**Comment by bhardwajaditya** [comment]
@ear-dev yeah just checked, that seems to be a problem for iframe cases

**Comment by bhardwajaditya** [comment]
@ear-dev touch detection fixed the issue

File: src/lib/logger.js [hunk 1]
```diff
@@ -2,7 +2,26 @@ import util from 'util';
...
+		} else if ('orientation' in window) {
+			hasTouchScreen = true;
+		} else {
+			const UA = navigator.userAgent;
+			hasTouchScreen = /\b(BlackBerry|webOS|iPhone|IEMobile)\b/i.test(UA)
+              || /\b(Android|Windows Phone|iPad|iPod)\b/i.test(UA);
+		}
+	}
+	return hasTouchScreen;
+};
```
Comment (ear-dev) [inline src/lib/logger.js:24]: Thanks @bhardwajaditya this worked when I tested it on my phone....  I see a lot of conditions here......  I assume it should work to detect all mobile devices?

Comment (bhardwajaditya) [reply src/lib/logger.js:24]: Yes it will work, this is based on mdn specifications

**Review by ear-dev — APPROVED** [review]

**Pull request merged**